addon/utils/prop-types.js

```diff
@@ -61,6 +61,10 @@ export function generateType (key) {
         def.required = options.required
       }
 
+      if (options.updatable === false) {
+        def.updatable = false
```

Comment:
Properties were always updatable so when `updatable` is omitted they can be updated. I opted not to add the key here when the value is true since it functions the same as not providing the option at all.